doc/manual/src/release-notes/rl-next.md

```diff
@@ -1,7 +1,8 @@
 # Release X.Y (202?-??-??)
 
-- The experimental nix command is now a `#!-interpreter` by appending the
-  contents of any `#! nix` lines and the script's location to a single call.
+- The experimental nix command can now act as [a `#!` interpreter](@docroot@/command-ref/new-cli/nix.md#shebang-interpreter)
```

Comment:
```suggestion
- The experimental `nix` command can now act as [shebang interpreter](@docroot@/command-ref/new-cli/nix.md#shebang-interpreter)
```

I think that should be written out, already for searchability.

Reply:
Good point. Fixed!